PowerShellTools/Resources.resx

```diff
@@ -405,4 +405,16 @@ Would you like to download now?</value>
   <data name="AttachErrorTitle" xml:space="preserve">
     <value>Error Attaching to Process</value>
   </data>
+  <data name="CleanupErrorMessage" xml:space="preserve">

```

Comment:
Hopefully we never get to this state, and eventually we have an option to restart the host process.

Regardless, " in order to restore the tools to a stable state." is redundant, and having 'detach/exit' is also redundant [Not actionable at all by the user]